src/styles/common.sss

```diff
@@ -9,7 +9,7 @@ html
 
 body
     font-size: 20px
-    font-family: Times New Roman, Times, serif
+    font-family: SourceSansPro, Times, serif
```

Comment:
This change will result in a merge conflict, the current `dev` branch also has changed the font family.